partner_event/models/res_partner.py

```diff
@@ -37,7 +37,7 @@ def _compute_registration_count(self):
     def _compute_event_count(self):
         for partner in self:
             partner.event_count = len(
-                self.env["event.registration"].search([
+                self.env["event.registration"].sudo().search([
```

Comment:
Please take the occasion to rewrite this to use read_group for performance reasons.

Reply:
This method is expected to be called only from form view (where this computed field is shown), so this doesn't seem a very critical performance point at first sight. Can't we leave that for the future?

Reply:
Not really. It's prefetched also on other situations. It's very simple, so please do it.